src/te/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -525,8 +548,13 @@ void InjectInline(ScheduleNode* sch, bool feature_extraction_mode) {
         for (auto iv : compute->axis) {
           args.push_back(iv->var);
         }
+        if (ext_ops.find(stage->op) != ext_ops.end()) {
+          // sshtin: The extern op can try to get access to the input tensors as a row data,
```

Comment:
sorry one more comment: Do you mean "raw data" here, or what is "row data" otherwise?